packages/ripple-nuxt-tide/lib/templates/axios.js

```diff
@@ -29,13 +30,15 @@ export default function ({ $axios, app, res }) {
     // Set http status code if a route or preview request failed.
     if (routeRequest || authPreviewRequest) {
       // We hide 403 and show it as 404
-      code = code === 403 ? 404 : code
-
-      app.tideResErrCode = code
-
-      if (typeof res !== 'undefined') {
-        res.statusCode = code
+      if (statusCode === 404 || statusCode === 403) {
+        message = 'Page not found'
+      } else if (statusCode > 499) {
+        message = 'Error fetching page'
       }
+      logger.info(`${message} - ${err.config.url}`, {label: 'Axios', requestId: err.config.headers['X-Request-Id']})
+    } else {
+      logger.info('Error fetching request', err.config.url, {label: 'Axios', requestId: err.config.headers['X-Request-Id']})
     }
+    return Promise.reject({ statusCode, message })
```

Comment:
We need to rethink non essential requests then, i dont like partially loading the page if a request fails. I think its a recipe for introducing a whole class of hard to diagnose bugs. If a menu fails I feel like we _should_ throw an error. What if its essential to be able to navigate to another page? Hiding it just means we dont fix the underlying issue, that might have been appropriate earlier in the project but i dont think it is now.

Agree we need tests for this, I was using `axios-mock-adapter` locally to test network timeouts, I might add that and tests for timeouts.

Excluding the issue with authenticated requests (401s) returning as 404, what would be the issue with returning the BE status code to the browser? It will help us diagnose the actual issue and the user will see the glitch message anyway.